packages/glade.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Glade < Package
+class Glade < Meson
   description 'User Interface Builder for GTK+ applications'
   homepage 'https://wiki.gnome.org/Apps/Glade'
   version '3.38.2'
```

Comment:
Update to [3.40.0](https://download.gnome.org/sources/glade/3.40/glade-3.40.0.tar.xz)?